app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+class Category < ApplicationRecord
+  has_many :books
```

Comment:
wtf? try products

Reply:
we shoul use here relation has_and_belongs_to_many

Reply:
lol sry, mb fixed that. relation has_and_belongs_to_many seems pointless to me since product belongs to category and it wont change none, but fs ill change it, hope u can explain why we need to use it